.github/workflows/container-build.yml

```diff
@@ -59,5 +60,23 @@ jobs:
       - name: creds
         run: gcloud auth configure-docker --quiet
 
+      - name: Formatted labels
+        id: labels
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          FORMATED_LABELS="--image-label commit-hash=$GITHUB_SHA"
+
```

Comment:
Do we need commit hash as a label? Looks like we include that as a tag already.

Reply:
if that does not hurt, it is an extra thing :)